management-interface.go

```diff
@@ -116,6 +116,33 @@ func getDeviceName() string {
 	return strings.SplitN(name, ".", 2)[0]
 }
 
+// Return the serial number for the Raspberr Pi in the device.
+func getRaspberryPiSerialNumber() string {
+
+	if runtime.GOOS == "windows" {
+		return ""
+	}
+
+	// Run /proc/cpuinfo command to get the info we want.
+	out, err := exec.Command("cat", "/proc/cpuinfo").Output()
+	if err != nil || len(out) == 0 {
+		return ""
+	}
```

Comment:
There's no need to shell out to a command to read a file here. As far as your program is concerned /proc/cpuinfo is just another file on the filesystem. Just use `ioutil.ReadAll`.